index.html

```diff
@@ -5132,6 +5132,12 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type MUST NOT be used with <dfn>embedded proof</dfn>.
```

Comment:
```suggestion
    This media type MUST NOT be used to describe a verifiable credential with an <dfn>embedded proof</dfn>.
```

Reply:
Again, I'm for removing this statement all together, it doesn't seem to be useful and having it seems unnecessarily restrictive and counterintuitive. A verifiable credential is *also* a credential. Whether it's safe to use is a matter of actually trying to verify its proof, not some metadata statement specifying a media type. We shouldn't imply otherwise.

Reply:
I like this statement +1.  When reading this suggested change as an implementer it tells me something very important about how I can handle the data for this media type, that I'm applying different rules to this content than I would if I'm expecting/requiring an embedded proof.

Reply:
@quartzjer,

Can you make that more concrete with an example? Then, in your example (of your choice), suppose I hand you a credential that has an embedded proof in one case and I hand you a credential that does not in another, but both are tagged with the same media type. What would you do differently and why?

Reply:
I'm merging https://github.com/w3c/vc-data-model/pull/1014#discussion_r1102128119

based on the support I can see for it.

Reply:
> When reading this suggested change as an implementer it tells me something very important about how I can handle the data for this media type, that I'm applying different rules to this content than I would if I'm expecting/requiring an embedded proof.

Yikes, no that's dangerous. If you are performing a security process by purely doing media type detection you're setting yourself up for failure... you need to check the security characteristics of the message because we have good data now that developers don't always get media types right. Purely switching off of media type with guidance along the lines of "you can trust this" or "you cannot trust this" enables attackers to have another attack vector. That should probably be what we convey in this media types guidance... that the media type is a decent signal, but you shouldn't be making security decisions purely off of it.